### PR TITLE
Feature/user view

### PR DESCRIPTION
**Pull request by Jldeatrick**
Adding profile page for each user, displaying relevant user data, including the engagements in which a user has participated.

### CODE REVIEW

**Review by kimhoang73 — APPROVED** [review]
looks good to me! i like how we added all the relevant data to the users in a user friendly way. providing direct links to all engagements, screencasts and presentations will also be useful to the user. also having the same edit, destroy and csv options is a good idea in case an admin wanted to search those engagements by first finding a user

File: features/step_definitions/user_view_steps.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Given /^I am on the user information page for "(.*)"$/ do |username|
+#     u = User.find_by_name(username)
+#     visit user_path(u)
+# end
+
+# Then /^I should be on the user information page for "(.*)"$/ do |username|
+#     u = User.find_by_name(username)
+#     expect(visit user_path(u))
+# end
```
Comment (julies642) [inline features/step_definitions/user_view_steps.rb]: This file's steps seem to be commented out. Is this file necessary or were you planning to use these somewhere?

File: features/user_view.feature [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+# Story ID: 153070288
+Scenario: The User profile page should have the correct information
+    Given I am on the user details page for "Joe Deatrick"
+    Then I should see "jldeatrick@protonmail.com"
+    And I should see "student"
+    And I should see "Chromaticats"
+    And I should see "QDGames"
+    And I should not see "ACElab"
+    And I should not see "Teamscope"
```
Comment (kyriakipapadimitriou) [inline features/user_view.feature]: extensive testing. thanks for checking that there should not be apps the user does not belong to/owns.

**Review by kyriakipapadimitriou — APPROVED** [review]
great implementation and tests. 
For future features folks could implement tests that check what happens to a profile of a user if you remove them from an app.

File: features/step_definitions/user_view_steps.rb (hunk 1 above)
Comment (csungwon) [reply features/step_definitions/user_view_steps.rb]: oh they have to be removed. Thanks for catching!
